### PR TITLE
update DCO instructions

### PR DESCRIPTION
**Pull request by wmorgan**
Signed-off-by: William Morgan <william@buoyant.io>

### CODE REVIEW

**Comment by wmorgan** [comment]
cc @caniszczyk

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -16,8 +16,10 @@ as a contributor, have the legal right to make the contribution.
 
 See the [DCO](DCO) file for the full text of what you must agree to.
 
-To signify that you agree to the DCO for a commit, you add a line to the git
-commit message:
+### Option 1: commit message signoffs
```
Comment (olix0r) [inline CONTRIBUTING.md:19]: markdown lint is complaining that these headers are inconsistent with the rest of the doc

```suggestion
### Option 1: commit message signoffs ###
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -27,6 +29,25 @@ In most cases, you can add this signoff to your commit automatically with the
 `-s` flag to `git commit`. You must use your real name and a reachable email
 address (sorry, no pseudonyms or anonymous contributions).
 
+### Option 2: public statement
```
Comment (olix0r) [inline CONTRIBUTING.md:32]:
```suggestion
### Option 2: public statement ###
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -27,6 +29,25 @@ In most cases, you can add this signoff to your commit automatically with the
...
+comment on the PR with the following statement: "I agree to the DCO for all the
+commits in this PR."
+
+Note that this option also requires that your commits are made under your real
+name and a reachable email address.
+
+If you use this approach, the DCO bot will still complain, but maintainers will
+override the DCO bot at merge time.
+
+### Option 3: very simple changes
```
Comment (olix0r) [inline CONTRIBUTING.md:45]:
```suggestion
### Option 3: very simple changes ###
```

**Review by olix0r — APPROVED** [review]

**Review by adleong — APPROVED** [review]

**Pull request merged**